### DISCUSSION TITLE
Empty probability map found with Label Sampler

### DISCUSSION THREAD

**Discussion by ghost** [TorchIO-project/torchio#871] [Q&A]

**Comment by fepegar** [comment]
Hi, @JonasZaoui. Instead of deleting your text, could you please share your solution?

**Comment by ghost** [comment]
Hello ! Of course sorry, 
I want segment microbleeds (very unbalanced data, only a few voxels per image), and I have this error in the majority of case when i plot randomly a patch, when i use LabelSampler.
```

Voxels with positive probability might be near the image border.
If you suspect that this is the case, try adding a padding transform
with half the patch size: torchio.Pad((64, 64, 16))

```
Here is some information :

    Size of Images : 208x208x48
    Patch of images : 128x128x32 

And my code :

```
sampler = tio.data.LabelSampler(self.patch_size, label_name='mask',)
self.max_queue_length = 100
self.samples_per_volume = 5
self.patches_training_set = tio.Queue(
                               subjects_dataset=self.training_set,
                               max_length=self.max_queue_length,
                               samples_per_volume=self.samples_per_volume,
                               sampler=sampler,
                               num_workers=self.num_workers,
                      )

```
So I tried to pad the images as indicated by the error and play with patch size. Its works now, thanks
